src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,50 +4,50 @@
     "origin.toomanyorigins.hare.name": "Hare",
 
     "origin.toomanyorigins.hisskin.name": "Hiss-kin",
-    "origin.toomanyorigins.hisskin.description": "Creeperın bu uzak akrabası, patlamalarını kaçış amaçlı kullanır.",
+    "origin.toomanyorigins.hisskin.description": "Creeper'ın akrabası olan Hiss-kin, patlamalarını kaçış amaçlı kullanır.",
   
     "origin.toomanyorigins.swarm.name": "Swarm",
-    "origin.toomanyorigins.swarm.description": "Swarm ırkı, çatışmadan uzak duran uzman çiftçi ve kovandan bağımsız böceklerdir.",
+    "origin.toomanyorigins.swarm.description": "Swarm ırkı, çatışmadan uzak duran, uzman çiftçi ve kovandan bağımsız böceklerdir.",
 
     "origin.toomanyorigins.undead.name": "Undead",
 
     "origin.toomanyorigins.withered.name": "Withered",
-    "origin.toomanyorigins.withered.description": "Withered ırkı, atalarının Wither'ın kaotik gücünden yararlanmalarının bir sonucu olarak öldürdüklerinin yaşam enerjisini emerler.",
+    "origin.toomanyorigins.withered.description": "Witheredlar, atalarının Wither'ın kaotik gücünden yararlanması sonucu öldürdüklerinin yaşam enerjisini emer.",
   
     "power.toomanyorigins.overheat.name": "Aşırı Isınma",
-    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni biraz incitir.",
-    "power.toomanyorigins.aerial_stability.name": "Hava Stabilliği",
+    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni incitir.",
+    "power.toomanyorigins.aerial_stability.name": "Hava Dengesi",
     "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızınız biraz azaltılmış.",
     "power.toomanyorigins.conductive.name": "İletken",
     "power.toomanyorigins.conductive.description": "Yıldırım tarafından çarpıldığında şarjlanırsın.",
     "power.toomanyorigins.stubby_legs.name": "Güdük Bacaklar",
-    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların ayakta daha yavaş hareket etmeni sağlar.",
+    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların yüzünden karada daha yavaş yürürsün.",
   
     "effect.toomanyorigins.charged": "Şarjlı",
     "death.attack.overheat": "%1$s içini fitilledi",
-    "death.attack.overheat.player": "%1$s, %2$s den kaçmaya çalışırken içini fitilledi.",
-    "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s den kaçmaya çalışırken içini fitilledi.",
+    "death.attack.overheat.player": "%1$s, %2$s'den kaçmaya çalışırken içini fitilledi.",
+    "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s'den kaçmaya çalışırken içini fitilledi.",
   
     "power.toomanyorigins.hover.name": "Süzül",
-    "power.toomanyorigins.hover.description": "Düşerken yükseklik kaybetmeden süzülebilirsiniz.",
+    "power.toomanyorigins.hover.description": "Düşeceğin zaman havada asılı kalabilirsin, bu şüreçte biraz acıkırsın.",
```

Comment:
now using "Stay still on air" instead of "Float without losing altitude" as an alternative to "Hover"(hover doesn't exist in turkish)